<!-- This is a launch file to control the triskar robot using a PS3-like Joystick. What is different here
     is the fact that I can increase velocity using the joystick not the keyboard as in the default
     joystick node ros tutorial.-->

<launch>

	<rosparam file="$(find joystick)/params/joystick_params.yaml" command="load" />
	
	<node pkg="joy" type="joy_node" name="joy">
		<param name="dev" type="string" value="/dev/input/js0" />
        <param name="autorepeat_rate" value="60" />
	</node>

	<node pkg="joystick" type="joystick_node" name="joystick_node" output="screen" respawn="false">
		<remap from="raw_cmd_vel" to="cmd_vel"/>
	</node>

</launch>
